src/uwtools/api/driver.py

```diff
@@ -161,5 +151,19 @@ def _get_driver_module_implicit(module: str) -> Optional[ModuleType]:
         return None
 
 
-__all__: list[str] = [graph.__name__]
-_add_classes()
+__all__ = [
+    getattr(x, "__name__")
+    for x in (
+        Assets,
+        AssetsCycleBased,
+        AssetsCycleLeadtimeBased,
+        AssetsTimeInvariant,
+        Driver,
+        DriverCycleBased,
+        DriverCycleLeadtimeBased,
+        DriverTimeInvariant,
+        execute,
+        graph,
+        tasks,
+    )
+]
```

Comment:
The docs currently on `main` are not showing `execute()` or `tasks()`. This update should ensure that all the current module contents are included.

Reply:
I agree, but I don't see a way around it: The old solution, using a single list of strings and dynamically importing and then building up `__all__` from that list had the unfortunate downside of obscuring the fact that these classes are members of the module. If anyone can think of a DRYer solution that doesn't reintroduce the visibility problem, I'm definitely interested.

Reply:
It's fine. Just generally voicing my dissent at Python/pylint over this.

Reply:
If the issue was only internal to `uwtools`, I'd stick with the dynamic-add behavior and `# pylint: disable` the complaint. But I think the duplication and maintenance burden thereof are worth it for a better user experience. I think we will -- as any good tool / library developer does -- find ourselves dealing with some painful complexity internally to create a (hopefully!) magical user experience.